sqlobject/src/main/java/org/jdbi/v3/sqlobject/SqlObjectFactory.java

```diff
@@ -43,11 +41,24 @@
 public enum SqlObjectFactory implements ExtensionFactory<SqlObjectConfig> {
     INSTANCE;
 
-    private static final MethodInterceptor NO_OP = (proxy, method, args, methodProxy) -> null;
+    private static final Object[] NO_ARGS = new Object[0];
+    private static final Method EQUALS_METHOD;
+    private static final Method HASHCODE_METHOD;
+    private static final Method TOSTRING_METHOD;
+
+    static {
+        try {
+            EQUALS_METHOD = Object.class.getMethod("equals", Object.class);
+            HASHCODE_METHOD = Object.class.getMethod("hashCode");
+            TOSTRING_METHOD = Object.class.getMethod("toString");
+        }
+        catch (NoSuchMethodException wat) {
+            throw new IllegalStateException("JVM error", wat);
+        }
+    }
```

Comment:
Sorry for a dumb question, but why do we need here handle `Object` methods? It looks like if we omit those lines, all tests pass. As I see, `OnDemandExtensions` already takes care of them.